Update documentation (#107)
OpenSCAP/OVAL-visualization-as-graph: docs/GUIDE.md

## Installation

### Enterprise Linux

> **Before install enable Extra Packages for Enterprise Linux (EPEL). How to enable EPEL is written [here](https://fedoraproject.org/wiki/EPEL).**

Reviewer: ```suggestion
**Before installing, enable Extra Packages for Enterprise Linux (EPEL). Learn how to enable EPEL in [EPEL documentation](https://fedoraproject.org/wiki/EPEL).**
```

arf-to-graph -o ~/graphs scan-data/ssg-fedora-ds-arf.xml 'xccdf_org.ssgproject.content_rule_audit_rules_unsuccessful_\w+on_creat'
```

> This command generates a graph and saves file named `graph-of-<rule_id>-<date>.html` (The date the graph was created.) in `~/graphs` because `-o` was used. Then open web browser. _Default web browser is Firefox. If Firefox not installed it opens default web browser in OS._

Reviewer: ```suggestion
This command generates a graph and saves file named `graph-of-<rule_id>-<date>.html` (The date the graph was created.) in `~/graphs` because `-o` was used. Then, it opens the generated file in your web browser. _Default web browser is Firefox. If Firefox is not installed, the default web browser in OS is used._
```
